Data/market-hours/market-hours-database.json

```diff
@@ -10225,7 +10225,81 @@
       "saturday": [],
       "holidays": [
         "4/14/2017"
-      ]
-    }
+      ]    
+    },
+        "Forex-bitfinex-[*]": {
+            "dataTimeZone": "UTC",
+            "exchangeTimeZone": "UTC",
+            "sunday": [
+                {
+                    "start": "00:00:00",
+                    "end": "1.00:00:00",
```

Comment:
Since submitting this I also realized that the exchange in question does not operate during Chinese new year, which might need addressing at some point. For this PR, I have just supplied what is necessary to define a new market for the purposes of backtesting using decimalized order quantities.  You're right that any non supported order types will be rejected with an exception, however, this is implemented in the brokerage integration.